src/rdkafka_transport.c

```diff
@@ -498,6 +500,14 @@ static int rd_kafka_transport_ssl_connect (rd_kafka_broker_t *rkb,
 	if (!SSL_set_fd(rktrans->rktrans_ssl, rktrans->rktrans_s))
 		goto fail;
 
+	mtx_lock(&rkb->rkb_lock);
+	rd_snprintf(name, sizeof(name), "%s", rkb->rkb_nodename);
```

Comment:
`rkb_origname` is more difficult to parse? Couldn't it be `ssl://nodename:9094` for example? What difference would it be compared to `rkb_nodename`?